src/main/java/core/basesyntax/DifferentColors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum DifferentColors {
```

Comment:
```suggestion
public enum Color {
```
I'd recommend naming it this way. Let's make your namings more concise and clear